openshift_sti_images/base.adoc

```diff
@@ -0,0 +1,48 @@
+= Base
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+Our OpenShift https://github.com/openshift/sti-base/tree/master/2.0[Base] image repository contains Dockerfiles for building base images. Base image contains all the essential libraries and tools needed for OpenShift language images, namely:
```

Comment:
I think this link has gone bad. Could just go to https://github.com/openshift/sti-base, yea?